tests/integration/motions/test_viewset.py

```diff
@@ -106,6 +106,22 @@ def test_with_tag(self):
         motion = Motion.objects.get()
         self.assertEqual(motion.tags.get().name, 'test_tag_iRee3kiecoos4rorohth')
 
+    def test_with_multiple_comments(self):
+        config['motions_comments'] = [
+            { 'name': 'comment1', 'public': True},
+            { 'name': 'comment2', 'public': False}]
+        comments = ['comemnt1_sdpoiuffo3%7dwDwW&', 'comment2_iusd&D/TdskDWH&5DWas46WAd078']
+        response = self.client.post(
+            reverse('motion-list'),
+            {'title': 'title_test_sfdAaufd56HR7sd5FDq7av',
+             'text': 'text_test_fiuhefF86()ew1Ef346AF6W',
+             'comments': comments})
+        for data in response:
+            print(repr(data))
```

Comment:
@normanjaeckel Why there is this error: `Data must ba an array.`? I send an array, but the serializer gets a string with the last comment..